readthedocs/templates/projects/environmentvariable_list.html

```diff
@@ -0,0 +1,45 @@
+{% extends "projects/project_edit_base.html" %}
+
+{% load i18n %}
+
+{% block title %}{% trans "Environment Variables" %}{% endblock %}
+
+{% block nav-dashboard %} class="active"{% endblock %}
+
+{% block editing-option-edit-environment-variables %}class="active"{% endblock %}
+
+{% block project-environment-variables-active %}active{% endblock %}
+{% block project_edit_content_header %}{% trans "Environment Variables" %}{% endblock %}
+
+{% block project_edit_content %}
+  <div class="button-bar">
+    <ul>
+      <li>
+        <a class="button"
+           href="{% url 'projects_environmentvariables_create' project_slug=project.slug %}">
+          {% trans "Add Environment Variable" %}
```

Comment:
What do we want to communicate on Environment Variable _list_ page to the user?

Reply:
I like Heroku's example. Just a short blob explaining env vars and point to the docs for anything substantial. I'm not big on prose content in UI.